File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
dice = str(input())
dice = list(map(int,dice.split()))

match (dice[0]-dice[1], dice[1]-dice[-1]):
    case (0,0):
        print(10000 + dice[0] * 1000)
    case (0,_):
        print(1000 + dice[0] * 100)
    case (_,0):
        print(1000 + dice[1] * 100)
    case (_,_):
        if dice[0] == dice[-1]:
            print(1000 + dice[0] * 100)
        else:
            var = max(dice)
            print(var * 100)
        
        
        

File: node.js/백준/Bronze/2480. 주사위 세개/주사위 세개.js
let rl = require('readline').createInterface({
    input: process.stdin,
});

let input;

rl.on('line', function(li) {
    input = li.toString().split(' ').map(el => parseInt(el));
    rl.close();
}).on('close', function() {
    dice(input);
    process.exit();
});

function dice(lst) {
    let arr = Array.from({length: 7}, ()=>0);
    for(let i of lst) {
        arr[i] += 1;
    }
    
    if (arr.includes(3)) {
        console.log(10000 + 1000 * arr.indexOf(3));
    } else if (arr.includes(2)) {
        console.log(1000 + 100 * arr.indexOf(2));
    } else {
        console.log(arr.lastIndexOf(1) * 100);
    }
}